partial-flashing.js

```diff
@@ -492,12 +492,20 @@ let PartialFlashing = {
                 if (window.previousDapWrapper) {
                     return window.previousDapWrapper.disconnectAsync()
                         .finally(() => {
-                                window.previousDapWrapper = null;
+                                // window.previousDapWrapper = null;
```

Comment:
Is this meant to be commented out? If so I guess we should remove the finally?

Reply:
Yep